sql/plsql/PlSqlParser.g4

```diff
@@ -6214,9 +6215,105 @@ string_function
+    | JSON_OBJECT '(' json_object_content ')'
+    | JSON_OBJECTAGG '(' KEY? expression VALUE expression ((NULL_ | ABSENT) ON NULL_)? (RETURNING ( VARCHAR2 ('(' UNSIGNED_INTEGER ( BYTE | CHAR )? ')')? | CLOB | BLOB ))?  STRICT? (WITH UNIQUE KEYS)?')'
+    | JSON_QUERY '(' expression (FORMAT JSON)? ',' CHAR_STRING json_query_returning_clause? json_query_wrapper_clause? json_query_on_error_clause? json_query_on_empty_clause? ')'
+    | JSON_SERIALIZE '(' CHAR_STRING (RETURNING json_query_return_type)? PRETTY? ASCII? TRUNCATE? ((NULL_ | ERROR | EMPTY (ARRAY | OBJECT)) ON ERROR)? ')'
+    | JSON_TRANSFORM '(' expression ',' json_transform_op (',' json_transform_op)* ')'
+    | JSON_VALUE '('  expression (FORMAT JSON)? (',' CHAR_STRING? json_value_return_clause? ((ERROR | NULL_ | DEFAULT literal)? ON ERROR)? ((ERROR | NULL_ | DEFAULT literal)? ON EMPTY)? json_value_on_mismatch_clause?')')?
+    ;
+
+json_object_content
+    : (json_object_entry (',' json_object_entry)* | '*') json_on_null_clause? json_return_clause? STRICT? (WITH UNIQUE KEYS)?
+    ;
+
+json_object_entry
+    : (KEY? expression (VALUE | IS)? expression | expression ':' expression (FORMAT JSON)?) | identifier
+    ;
+
+json_table_clause
+    : JSON_TABLE '(' expression (FORMAT JSON)? (',' CHAR_STRING)? ((ERROR | NULL_) ON ERROR)? ((EMPTY | NULL_) ON EMPTY)? json_column_clause? ')'
+    ;
+
+json_array_element
+    : (expression | CHAR_STRING | NULL_ |  UNSIGNED_INTEGER | json_function) (FORMAT JSON)?
+    ;
+
+json_on_null_clause
+    : (NULL_ | ABSENT) ON NULL_
+    ;
+
+json_return_clause
+    : RETURNING ( VARCHAR2 ('(' UNSIGNED_INTEGER ( BYTE | CHAR )? ')')? | CLOB | BLOB )
+    ;
+
+json_transform_op
+    : REMOVE CHAR_STRING ((IGNORE | ERROR)? ON MISSING)?
+    | INSERT CHAR_STRING '=' CHAR_STRING ((REPLACE | IGNORE | ERROR) ON EXISTING)? ((NULL_ | IGNORE | ERROR | REMOVE)? ON NULL_)?
+    | REPLACE CHAR_STRING '=' CHAR_STRING ((CREATE | IGNORE | ERROR) ON MISSING)? ((NULL_ | IGNORE | ERROR)? ON NULL_)?
+    | expression (FORMAT JSON)?
+    | APPEND CHAR_STRING '=' CHAR_STRING ((CREATE | IGNORE | ERROR) ON MISSING)? ((NULL_ | IGNORE | ERROR)? ON NULL_)?
+    | SET CHAR_STRING '=' expression (FORMAT JSON)? ((REPLACE | IGNORE | ERROR) ON EXISTING)? ((CREATE | IGNORE | ERROR) ON MISSING)? ((NULL_ | IGNORE | ERROR)? ON NULL_)?
+    ;
+
+json_column_clause
+    : COLUMNS '(' json_column_definition (',' json_column_definition)* ')'
+    ;
+
+json_column_definition
+    : expression json_value_return_type? EXISTS? PATH CHAR_STRING json_query_on_error_clause? json_query_on_empty_clause?
+    | expression json_query_return_type? TRUNCATE? FORMAT JSON  json_query_wrapper_clause? PATH CHAR_STRING json_query_on_error_clause? json_query_on_empty_clause?
+    | expression json_value_return_type? TRUNCATE? (PATH CHAR_STRING)? json_query_on_error_clause? json_query_on_empty_clause?
```

Comment:
It looks like it can be simplified in the following way:

```suggestion
    : expression json_value_return_type? (EXISTS? PATH CHAR_STRING | TRUNCATE (PATH CHAR_STRING)?)? json_query_on_error_clause? json_query_on_empty_clause?
    | expression json_query_return_type? TRUNCATE? FORMAT JSON  json_query_wrapper_clause? PATH CHAR_STRING
```